### ISSUE TITLE
Why are the 'bright' ANSI colors darker?

### ISSUE THREAD

**Issue by jasoneveleth** [joshdick/onedark.vim#232]
I love your color scheme! I am curious as to why the 16 terminal colors have darker versions as "brighter". I'm thinking about switching all the brighter and darker versions (in my terminal) and then overriding them in vim, and I was wondering if there is a reason that I'm missing as to why this wasn't done. Thanks!

**Comment by joshdick** [comment]
Great question! I went back and forth on this when I was first putting this theme together.

tl;dr: Personal preference.

This was my thinking:

* At the time I made the decision, the original One Dark color palette didn't have corresponding dark colors for every regular color, really just yellow and red.
* The lower ANSI colors ("normal", 0-8) are personally what I prefer to see in the terminal for day-to-day usage, and most programs that I saw use red for error messages, etc, were using normal red and not "bright" red.
* So, because the "normal" colors were the ones I wanted to see most often and were used more frequently by programs in my own experience, and because most colors were missing dark versions in the palette, I "demoted" the darker colors to the less-commonly-used ANSI "bright color" positions.

In retrospect, I probably should have just repeated all of the colors for the 16-color palette and sacrificed the darker yellow/red.

**Comment by joshdick** [comment]
Closing this issue now that I've answered the question and there's no followup action needed.
